commons-test/src/main/java/org/infinispan/commons/test/annotation/TestForIssue.java

```diff
@@ -21,6 +21,13 @@
     *
     * @return The JIRA issue keys
     */
-   String[] jiraKey();
+   String[] jiraKey() default "";
+
+   /**
+    * The keys of the GitHub issues tested.
+    *
+    * @return The GitHub issue keys
+    */
+   String[] githubKey() default "";
```

Comment:
It looks like you're the only one using this annotation :smile:

Reply:
I never even knew this existed! Also I guess it needs to be updated as we aren't using JIRA anymore.

Reply:
Yes I stealed it from the Hibernate team. The problem is that we don't have GitHub references for old issues.